docs/src/main/asciidoc/security-proactive-authentication.adoc

```diff
@@ -104,11 +104,11 @@ For example:
 
 [source,java]
 ----
-package io.quarkus.it.keycloak;
```

Comment:
Should we keep the package ?

Reply:
My reasoning is that previous code snipets in this guide not using any package and in my POV at least in one guide the include/exclude of package in snipet should be same. Also sometimes when copy the code and pasting it to my ide the change of package is needed.

Maybe it's only me who prefer not having package names in examples if not necessary

If you want I revert this change back.

Reply:
@jedla97 I see what you mean, but then we'd also like these Java fragments be as complete Java files as possible, we seem to have packages everywhere like in https://quarkus.io/guides/http-reference, etc

Reply:
Ok I changed it back